pgbot/commands/user.py

```diff
@@ -1011,3 +1011,98 @@ def filter_func(x):
         )
 
         await page_embed.mainloop()
+
+    async def cmd_stream(self):
+        """
+        ->type Other commands
+        ->signature pg!stream
+        ->description Show the ping-stream-list
+        Send an embed with all the users currently in the \
+        ping-stream-list
+        """
+        ping = db.DiscordDB("stream")
+        data = await ping.get([])
+        await embed_utils.replace(
+            self.response_msg,
+            "Memento ping list",
+            "\n".join((f"<@{user}>" for user in data)),
+        )
+
+    async def cmd_stream_addme(self):
+        """
+        ->type Other commands
+        ->signature pg!stream_addme
+        ->description Add yourself to the stream-ping-list
+        ->extended description
+        Add yourself to the stream-ping-list. You can always delete \
+        you later with `pg!strem_delme`
+        """
+        ping = db.DiscordDB("stream")
+        data = await ping.get([])
+        user = self.author.id
+        if user not in ping:
+            data.append(user)
+        else:
+            raise BotException(
+                "You are already in the list",
+                "How can I add you to the list if you are already there!?",
+            )
+        await ping.write(data)
+        await embed_utils.replace(
+            self.response_msg,
+            "Memento ping list",
+            "\n".join((f"<@{user}>" for user in data)),
+        )
+
+    async def cmd_stream_delme(self):
+        """
+        ->type Other commands
+        ->signature pg!stream_delme
+        ->description Remove yourself from the stream-ping-list
+        ->extended description
+        Remove yourself from the stream-ping-list. You can always add \
+        you later with `pg!strem_addme`
+        """
+        ping = db.DiscordDB("stream")
+        data: list = await ping.get([])
+        user = self.author.id
+        if user in ping:
+            data.remove(user)
+        else:
+            raise BotException(
+                "You are not in the list",
+                "How can I delete you from the list if"
+                "you are not there already!?",
+            )
+        await ping.write(data)
+        await embed_utils.replace(
+            self.response_msg,
+            "Memento ping list",
+            "\n".join((f"<@{user}>" for user in data)),
+        )
+
+    async def cmd_stream_ping(
+            self,
+            message: str = None
```

Comment:
str typehint will not work here, because it will take only the first word argument. You need to use `String` here, and make user insert data in quotes